submissions/chizaramOpara/html/Shopnownow.html

```diff
@@ -0,0 +1,53 @@
+	<!DOCTYPE html>
+	<html>
+	<head>
+		<meta charset= "UTF8">
+		<title>Shopnownow</title>
+	</head>
+		<body>  
+		<header><b> Shopnownow Ng </b></header>
+		<div>
+		<em>Shopping Made Easier</em>
+		<p> <b> Hurray! Our Christmas sales is ongoing. Free delivery is available for purchases above $1,000 </b> </p>
+		</div>
+		<div>
+		<img src="C:/Users/user/Pictures/shopnownow.jpg" width="150"/>
```

Comment:
This path is absolute and not relative. imaging I cloned your task, it, therefore, means that I can't view this image on my browser because I don't have this path (`C:/Users/user`) as you do